omaha/main.scons

```diff
@@ -65,19 +65,19 @@ _is_google_update_build = False
 # These values must be changed when customizing open source Omaha.
 # You may also wish to replace the language-neutral constants in the individual
 # *generated_resources_*.rc files with localized strings.
-_FULL_COMPANY_NAME = 'OmahaCompanyName Inc.'
-_SHORT_COMPANY_NAME = 'OmahaCompanyName'
+_FULL_COMPANY_NAME = 'C the Signs'
```

Comment:
the full name is `C the Signs Limited`

Reply:
done but I changed it to C the Signs Limited, https://www.gov.uk/government/organisations/companies-house